pkg/placement/placement.go

```diff
@@ -182,16 +184,16 @@ func NewPlacementService(opts PlacementServiceOpts) *Service {
 // Blocks until the service is closed and all connections are drained.
 func (p *Service) Run(ctx context.Context, listenAddress, port string) error {
 	if p.closed.Load() {
-		return errors.New("placement service is closed")
+		return apiErrors.PlacementServiceIsClosedOnRun("placement service is closed")
```

Comment:
The errors in the Start function are not part of the API, and we don't need to transform them into richer grpc errors. @JoshVanL would you agree?

Reply:
Yes, that's correct. We don't and should have a typed API error here.